djangocms_versioning/monkeypatch/extensions.py

```diff
@@ -0,0 +1,26 @@
+from cms.extensions.extension_pool import ExtensionPool
+from cms.models.titlemodels import PageContent
+
+
+def _copy_title_extensions(self, source_page, target_page, language, clone=False):
+    # the line below has been modified to accomodate versioning.
+    # import pdb; pdb.set_trace()
+    source_title = PageContent._original_manager.filter(
+        page=source_page, language=language
+    ).first()
+    if target_page:
+        # the line below has been modified to accomodate versioning.
+        target_title = PageContent._original_manager.filter(
```

Comment:
Looking at the original commit is there a clever way to change "pagecontent_set"? If there was out monkey patch would literally just change the set returned. Food for thought.

Reply:
I will leave this as it is for now.

Reply:
Ok. I don't have a solution for that so fine to leave.